### PR TITLE
[tfldump] Support to dump metadata in Model

### PR DESCRIPTION
**Pull request by llFreetimell**
If we dump `.tflite` file, there was always one dummy buffer with some values.
It was turned out to `metadata` in `Model`.
This commit will enable supporting to dump `metadata` in `Model`.

ONE-DCO-1.0-Signed-off-by: Seok NamKoong <seok9311@naver.com>

### CODE REVIEW

**Review by mhs4670go — DISMISSED** [review]
LGTM

**Comment by llFreetimell** [comment]
@nnfw-bot test-nncc-release

**Comment by llFreetimell** [comment]
@nnfw-bot test nncc-release

File: compiler/tfldump/src/Dump.cpp [hunk 1]
```diff
@@ -382,6 +383,14 @@ void dump_model(std::ostream &os, const tflite::Model *model)
   }
   os << std::endl;
 
+  // dump metadata
+  os << "metadata : B(index) name" << std::endl;
```
Comment (seanshpark) [inline compiler/tfldump/src/Dump.cpp:387]: Maybe surround with `if (metadata != nullptr) {` ?

Comment (seanshpark) [reply compiler/tfldump/src/Dump.cpp:387]: I've checked with old inception_v3_2018_04_27.tgz which doesn't seem to have metadata; _tfldump_ just exits without any message.

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by jinevening — APPROVED** [review]
LGTM

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**